### ISSUE TITLE
meet a warning:Failed prop type: Required prop `onRequestClose` was not specified in `Modal`.

### ISSUE THREAD

**Issue by greatzues** [oblador/react-native-lightbox#65]
A month ago when I used haven't happened this warning, yesterday I review my pic list component, this warning alert, again and again, no matter I try to catch the warning, not work.

### warning as follow
```react-native
02-13 00:20:08.724 2102-31244/com.electromechanicalfamily W/ReactNativeJS: Warning: Failed prop type: Required prop `onRequestClose` was not specified in `Modal`.
                                                                               in Modal (created by LightboxOverlay)
                                                                               in LightboxOverlay (created by Lightbox)
                                                                               in RCTView (created by View)
                                                                               in View (created by Lightbox)
                                                                               in Lightbox (created by StaticRenderer)
                                                                               in RCTView (created by View)
                                                                               in View (created by StaticRenderer)
                                                                               in StaticRenderer (created by ListView)
                                                                               in RCTView (created by View)
                                                                               in View (created by ScrollView)
                                                                               in RCTScrollView (created by ScrollView)
                                                                               in ScrollView (created by ListView)
                                                                               in ListView (created by StaticRenderer)
                                                                               in RCTView (created by View)
                                                                               in View (created by StaticRenderer)
                                                                               in RCTView (created by View)
                                                                               in View (created by StaticRenderer)
                                                                               in StaticRenderer (created by ListView)
                                                                               in RCTView (created by View)
                                                                               in View (created by ScrollView)
                                                                               in RCTScrollView (created by ScrollView)
                                                                               in ScrollView (created by ListView)
                                                                               in ListView (created by ListView)
                                                                               in ListView (created by Test)
                                                                               in Test (created by BottomTap)
                                                                               in TabNavigatorItem (created by BottomTap)
                                                                               in StaticContainer (created by SceneContainer)
                                                                               in RCTView (created by View)
                                                                               in View (created by SceneContainer)
                                                                               in SceneContainer (created by TabNavigator)
                                                                               in RCTView (created by View)
                                                                               in View (created by TabNavigator)
                                                                               in TabNavigator (created by BottomTap)
                                                                               in RCTView (created by View)
                                                                               in View (created by DrawerLayoutAndroid)
                                                                               in AndroidDrawerLayout (created by DrawerLayoutAndroid)
                                                                               in DrawerLayoutAndroid (created by BottomTap)
                                                                               in BottomTap (created by Navigator)
                                                                               in RCTView (created by View)
                                                                               in View (created by Navigator)
                                                                               in RCTView (created by View)
                                                                               in View (created by Navigator)
                                                                               in RCTView (created by View)
                                                                               in View (created by Navigator)
                                                                               in Navigator (created by ElectromechanicalFamily)
                                                                               in ElectromechanicalFamily (created by Root(ElectromechanicalFamily))
                                                                               in StaticContainer (created by Root(ElectromechanicalFamily))
                                                                               in RCTView (created by View)
                                                                               in View (created by Root(ElectromechanicalFamily))
                                                                               in Root(ElectromechanicalFamily)
                                                                               in RCTView (created by View)
                                                                               in View (created by AppContainer)
                                                                               in RCTView (created by View)
                                                                               in View (created by AppContainer)
                                                                               in AppContainer
```

### my code
```react-native
export default class LittleGround extends Component{
    renderScene(route, navigator) {
        var Component = route.component;

        return (
            <Component navigator={navigator} route={route} {...route.passProps} />
        );
    }

    render() {
        return (
            <Navigator
                ref="navigator"
                style={{ flex: 1 }}
                renderScene={this.renderScene.bind(this)}
                initialRoute={{
                    component: Test,
                }}
            />
        );
    }
}

.....

 picList(rowData, sectionID, rowID){
        var picUri = BASEURL+'/message/'+rowData;
        try{
            if(rowData!==null){
                return (
                    <View>
                        <Lightbox style={styles.itemContainer} springConfig={{tension: 50, friction: 5}} renderContent={this.renderCarousel.bind(this,picUri)}>
                            <Image
                                style={styles.imageItem}
                                source={{uri:picUri}}
                                resizeMode={'cover'}
                            />
                        </Lightbox>
                    </View>
                );
            }
        }catch(e){
            console.log(e);
        }
        return null
    }

    renderCarousel(uri) {
        return (
            <Carousel style={{ width: device.width, height: device.height }}>
                <Image
                    style={{flex: 1}}
                    resizeMode="contain"
                    source={{ uri: uri }}
                />
            </Carousel>
        );
    }
```
Everything goes well, but the warning can't ne solved.

**Comment by alfian0** [comment]
Same to me ...

**Comment by deathemperor** [comment]
check this PR https://github.com/oblador/react-native-lightbox/pull/51

**Comment by oblador** [comment]
Fixed in [0.7.0](https://github.com/oblador/react-native-lightbox/releases/tag/v0.7.0)
